src/mlagility/cli/cli.py

```diff
@@ -288,20 +287,21 @@ def main():
     # Parser for the "cache report" command
     #######################################
 
-    report_parser = cache_subparsers.add_parser(
-        "report", help="Generate reports in CSV format"
-    )
-    report_parser.set_defaults(func=report.summary_spreadsheet)
-
-    report_parser.add_argument(
-        "-d",
-        "--cache-dir",
-        dest="cache_dir",
-        help="The reports will be generated based on the builds in this GroqFlow "
-        f"build cache directory (defaults to {filesystem.DEFAULT_CACHE_DIR})",
-        required=False,
-        default=filesystem.DEFAULT_CACHE_DIR,
-    )
+    # TODO: https://github.com/groq/mlagility/issues/142
+    # report_parser = cache_subparsers.add_parser(
+    #     "report", help="Generate reports in CSV format"
+    # )
+    # report_parser.set_defaults(func=report.summary_spreadsheet)
+
+    # report_parser.add_argument(
+    #     "-d",
+    #     "--cache-dir",
+    #     dest="cache_dir",
+    #     help="The reports will be generated based on the builds in this GroqFlow "
+    #     f"build cache directory (defaults to {filesystem.DEFAULT_CACHE_DIR})",
+    #     required=False,
+    #     default=filesystem.DEFAULT_CACHE_DIR,
+    # )
```

Comment:
The message `Use the 'report' command to get a .csv file that summarizes results across all builds in the cache` is still being shown.

Reply:
Done!